scss/buttons.scss

```diff
@@ -0,0 +1,126 @@
+.btn {
```

Comment:
let's move this file to under `/foundation-style` if the code here was copied from foundation site